packages/schema/src/plugins/prisma/schema-generator.ts

```diff
@@ -584,7 +584,7 @@ export class PrismaSchemaGenerator {
                 'FieldReference',
                 new PrismaFieldReference(
                     resolved(node.target).name,
-                    node.args.map((arg) => new PrismaFieldReferenceArg(arg.name, this.exprToText(arg.value)))
+                    node.args.map((arg) => new PrismaFieldReferenceArg(arg.name, this.exprToText(arg.$container)))
```

Comment:
@ymc9 I tried to fix the typing by updating it. Could you check it please ? I will revert my change to make the tests pass.

Reply:
Sorry, it's ok. My editor complained about this and not anymore... very weird.